cocos2d/core/platform/CCSys.js

```diff
@@ -660,6 +666,9 @@ else {
     if (CC_EDITOR) {
         sys.isMobile = false;
         sys.platform = sys.EDITOR_PAGE;
+    } 
+    else if (CC_FB_PLAYABLE_ADS) {
+        sys.platform = sys.FB_PLAYABLE_ADS;
```

Comment:
Can you use `if (typeof FbPlayableAd !== undefined) { sys.platform = sys.FB_PLAYABLE_ADS; }` directly.